src/components/LHNOptionsList/OptionRowLHN.tsx

```diff
@@ -51,6 +52,15 @@ function OptionRowLHN({reportID, isFocused = false, onSelectRow = () => {}, opti
     const [isOnboardingCompleted = true] = useOnyx(ONYXKEYS.NVP_ONBOARDING, {
         selector: hasCompletedGuidedSetupFlowSelector,
     });
+    const [introSelected] = useOnyx(ONYXKEYS.NVP_INTRO_SELECTED);
+    const session = useSession();
+
+    // A guide is assigned to the user if they choose the 'MANAGE_TEAM' onboarding option, except for users with emails containing '+'.
```

Comment:
@c3024 I was thinking of something a bit more concise like "Guides are assigned for the `MANAGE_TEAM` onboarding action, except for emails that have `+`.". I think the rest is self-explanatory/easy to understand from a quick read of the code.

Reply:
Done!